src/utils/i18n-typed.ts

```diff
@@ -20,6 +20,8 @@ interface Options {
     ns: string; // namespace
     nsSeparator: string | boolean; // By default, ":", which breaks strings containing that char
     lng: string; // language
+    // useful to escape special characters in the translation string
+    interpolation: { escapeValue: boolean };
```

Comment:
https://www.i18next.com/translation-function/interpolation#unescape

Cool. I wasn’t familiar with this family of options. I typically use {{-var}}, but this way is fine and even more declarative (although it applies to the entire namespace).